integration-tests/deployment/ccip/deploy_home_chain.go

```diff
@@ -164,10 +172,8 @@ func AddChainConfig(
 	ccipConfig *ccip_config.CCIPConfig,
 	chainSelector uint64,
 	p2pIDs [][32]byte,
-	f uint8,
 ) (ccip_config.CCIPConfigTypesChainConfigInfo, error) {
 	// Need to sort, otherwise _checkIsValidUniqueSubset onChain will fail
-	sortP2PIDS(p2pIDs)
```

Comment:
sorting these once in the helper

Reply:
Nice!

Reply:
Sorting no longer needed, can remove it entirely